src/modules/75-cf/components/FlagArchiving/useArchiveFlagDialog.tsx

```diff
@@ -95,9 +101,39 @@ const useArchiveFlagDialog = ({
     STATUS.ok
   ])
 
-  const handleArchive = async (): Promise<void> => {
+  const { gitSyncInitialValues, gitSyncValidationSchema } = gitSync.getGitSyncFormMeta(
+    GIT_COMMIT_MESSAGES.ARCHIVED_FLAG
+  )
+
+  const [showGitModal, hideGitModal] = useModalHook(() => {
+    return (
+      <SaveFlagToGitModal
+        flagName={flagData.name}
+        flagIdentifier={flagData.identifier}
+        gitSyncInitialValues={gitSyncInitialValues}
+        gitSyncValidationSchema={gitSyncValidationSchema}
+        onSubmit={handleArchive}
+        onClose={hideGitModal}
+      />
+    )
+  }, [flagData.name, flagData.identifier, gitSync])
+
+  const handleArchive = async (gitSyncFormValues?: GitSyncFormValues): Promise<void> => {
+    let commitMsg = ''
```

Comment:
_TOTALLY NOT AN ISSUE_, but you don't need to initialise this to an empty string since you immediately reassign it.
```suggestion
    let commitMsg
```